solution/ui/regulations/eregs-vite/src/views/PolicyRepositorySearch.vue

```diff
@@ -192,6 +192,7 @@ if (!_isEmpty(searchQuery.value)) {
                         label="Search Unpublished Documents"
                         page="policy-repository-search"
                         :search-query="searchQuery"
+                        :show-suggestions=false
```

Comment:
to pass in a prop as a true Boolean type, you need to use `v-bind:` on the prop. We're using the `v-bind:` shorthand here (and most places), which is just a prepended colon (`:`).  TIL!